src/components/MultiGallery/useCreateGallery.ts

```diff
@@ -0,0 +1,59 @@
+import { useRouter } from 'next/router';
+import { useCallback } from 'react';
+import { graphql } from 'relay-runtime';
+
+import { useCreateGalleryMutation } from '~/generated/useCreateGalleryMutation.graphql';
+import { usePromisifiedMutation } from '~/hooks/usePromisifiedMutation';
+
+export default function useCreateGallery() {
+  const router = useRouter();
+
+  const [createGallery] = usePromisifiedMutation<useCreateGalleryMutation>(graphql`
+    mutation useCreateGalleryMutation($input: CreateGalleryInput!) @raw_response_type {
+      createGallery(input: $input) {
+        ... on CreateGalleryPayload {
+          gallery {
+            id
+            dbid
+          }
+        }
+
+        ... on ErrInvalidInput {
+          __typename
+        }
+      }
+    }
+  `);
+
+  return useCallback(
+    async (position: string) => {
+      try {
+        const response = await createGallery({
+          variables: {
+            input: {
+              name: '',
+              description: '',
+              position,
+            },
+          },
+        });
+
+        if (response?.createGallery?.__typename === 'ErrInvalidInput') {
+          return;
```

Comment:
we probably want to tell the user about this

Reply:
Added toast and throw error